sync/src/main/java/tech/pegasys/teku/sync/forward/multipeer/BatchSync.java

```diff
@@ -67,15 +72,19 @@
    */
   private UInt64 lastImportTimerStartPointSeconds;
 
+  private boolean scheduledSync = false;
```

Comment:
I'd use a more descriptive name, since the general guideline in teku is to have code self-explanatory, minimising comments.